lib/cadet/accounts/luminus.ex

```diff
@@ -47,11 +74,11 @@ defmodule Cadet.Accounts.Luminus do
       {:ok, "12742174091894830298409823098098"}
   """
 
-  def fetch_luminus_token!(nil) do
+  def fetch_luminus_token_or_return_default(nil) do
```

Comment:
Usually the way someone does it is:

```elixir
def something!(args) do
  {:ok, result} = something(args)
  result
end
```

Because this will raise `MatchError` if result is bad.